src/YouTubePlayer.js

```diff
@@ -31,18 +31,20 @@ YouTubePlayer.proxyEvents = (emitter) => {
  * Delays player API method execution until player state is ready.
  *
  * @todo Proxy all of the methods using Object.keys.
+ * @todo See TRICKY below.
  * @param {Promise} playerAPIReady Promise that resolves when player is ready.
  * @returns {Object}
  */
 YouTubePlayer.promisifyPlayer = (playerAPIReady) => {
   const functions = {};
 
   _.forEach(functionNames, (functionName) => {
-    functions[functionName] = (...args) => {
-      return playerAPIReady
-                .then((player) => {
-                  return player[functionName](...args);
-                });
+    functions[functionName] = async(...args) => {
```

Comment:
ESLint did not complain, but I can try to configure my IDE auto-format to correct this. I prefer a space there too.